src/Tickets/Seating/Assets.php

```diff
@@ -146,9 +147,10 @@ private function register_service_bundle(): void {
 
 			return [
 				'service' => [
-					'baseUrl'        => $this->container->get( Service\Service::class )->get_frontend_url(),
-					'mapsHomeUrl'    => $maps_layouts_home_page->get_maps_home_url(),
-					'layoutsHomeUrl' => $maps_layouts_home_page->get_layouts_home_url(),
+					'baseUrl'             => $this->container->get( Service\Service::class )->get_frontend_url(),
+					'mapsHomeUrl'         => $maps_layouts_home_page->get_maps_home_url(),
+					'layoutsHomeUrl'      => $maps_layouts_home_page->get_layouts_home_url(),
+					'associatedEventsUrl' => add_query_arg( [ 'page' => Associated_Events::get_slug() ], admin_url( 'admin.php' ) ),
```

Comment:
This change did not break a test or require a snapshot update, this means there is no coverage for it.
Add a test to cover this.